packages/webview_flutter/webview_flutter/example/lib/simple_example.dart

```diff
@@ -43,7 +43,8 @@ class _WebViewExampleState extends State<WebViewExample> {
           },
         ),
       )
-      ..loadRequest(Uri.parse('https://flutter.dev'));
```

Comment:
Is this for testing purposes? And should this be merged to main?

Reply:
I used it also for testing but it is more to showcase the user how he could make use of the newly added functionality, but not in this case. I tested in the android example, but here it should remain flutter.dev.